### PR TITLE
Feature modularize functions

### PR DESCRIPTION
**Pull request by laurenashley**
Moved helpers to class, output mean costs from analyse_costs class method

### CODE REVIEW

File: us-medical-insurance-costs-checkpoint.ipynb [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
```
Comment (PHWade) [inline us-medical-insurance-costs-checkpoint.ipynb:18]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Display IQR, Standard Deviation and Outliers for each region
</pre><p>This should be changed to make use of the new modular class method, just like in the bmi cost analysis part further down.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laurenashley/US-Medical-Insurance-Costs/pull/4/discussion/#comment-1209514146'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (PHWade) [inline us-medical-insurance-costs-checkpoint.ipynb:18]: <pre class="ql-syntax" spellcheck="false">Line #11.        print(f"median: {median}")
</pre><p>Super minor nitpick: missing dollar signs in median cost (which should be called median cost just for the benefit of the hypothetical people for whom this is intended)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laurenashley/US-Medical-Insurance-Costs/pull/4/discussion/#comment-1209514147'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='25'/>

Comment (laurenashley) [reply us-medical-insurance-costs-checkpoint.ipynb:18]: <p>Sounds good, will do</p>

Comment (laurenashley) [reply us-medical-insurance-costs-checkpoint.ipynb:18]: <p>agreed, I'll take care of it now</p>

Comment (laurenashley) [reply us-medical-insurance-costs-checkpoint.ipynb:18]: <p>Lookign into this further, we are expecting tuples from a dict being sent to analyse_costs but regions were structured differently making each region contain many single cost vals in the dict. We are however already making use of the class helper methods. </p>

Comment (PHWade) [reply us-medical-insurance-costs-checkpoint.ipynb:18]: So does that mean I need to define the various dictionaries differently when making use of the methods, or have you already addressed that? (Sorry I'm a bit confused)

**Pull request merged**